scripts/install-chef-suse.sh

```diff
@@ -911,7 +911,7 @@ fi
 
 # Use current time zone
 (
-    . /etc/sysconfig/clock || : Ignoring missing /etc/sysconfig/clock
+    TIMEZONE=$(LC_ALL=C timedatectl status | grep "zone:" | cut -d: -f2 | cut -d' ' -f2)
```

Comment:
awk is not installed, but I'm happy you found a place to bikeshed on. we can use the sed solution, I don't really care